hardware/schematic.md

```diff
@@ -13,21 +13,23 @@ review-with: jonnya
 
 ## Overview
 
-This page discusses the micro:bit schematic and Bill of Materials (BOM), which shows the electrical connections of the micro:bit and the components used in it.
+This page discusses the micro:bit schematic and Bill of Materials [BOM](https://github.com/microbit-foundation/microbit-v2-hardware/blob/main/V2/Bill.of.Materials-BBC-microbit_V2.0.0.csv), which shows the electrical connections of the micro:bit and the components used in it.
 
-The schematic is available from the BBC's [micro:bit hardware repository](https://github.com/bbcmicrobit/hardware)
+The micro:bit V1.3 and V1.5 schematic is available from the BBC's [micro:bit hardware repository](https://github.com/bbcmicrobit/hardware).
```

Comment:
We should probably add the CSS classes that distinguish the version numbers

```suggestion
The micro:bit <span class="v1">V1.3</span> and  <span class="v1">V1.5</span> schematic is available from the BBC's [micro:bit hardware repository](https://github.com/bbcmicrobit/hardware).
```

Reply:
I don't know what CSS classes are, so agree with your judgement